test/functional/apps/dashboard/dashboard_filter_bar.js

```diff
@@ -185,6 +192,9 @@ export default function ({ getService, getPageObjects }) {
     describe('saved search filtering', function () {
       before(async () => {
         await filterBar.ensureFieldEditorModalIsClosed();
+        await PageObjects.common.navigateToApp('discover');
+        await PageObjects.discover.switchDiscoverTable('new');
```

Comment:
not for now, but something we can do next time is to make this switch just update the local storage directly instead of clicking buttons :)